### PR TITLE
#257 Add geo object and instancing example with low poly 3d objects

### PR DESCRIPTION
**Pull request by pavletto**
Create geo object example with 3d low poly models
Fix instancing in GO. (crack it in #446 :()
Improve performance in recalculating indices! 

#### **Verify that...**

- [x] `npm test` passes
- [x] `npm run api` passes
- [x] `npm run lint` passes
- [x] `npm run build` still works

### CODE REVIEW

File: examples/geoObject/geoObject.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    terrain: new GlobusTerrain()
+});
+let colors = ["red", "orange", "yellow", "green", "lightblue", "darkblue", "purple"];
+
+let geoObjects = new EntityCollection({
+    entities: [],
+    scaleByDistance: [600000, 24000000, 10000000000]
+});
+
+const instancesTypes = [...new Array(80).fill('airplane'), ...new Array(20).fill('farmplane'), 'satellite'];
```
Comment (Zemledelec) [inline examples/geoObject/geoObject.js]: I do not like this iteration logic. I suggest replace it with something more convenient:
```
[
  {
      instanceName: "airplane",
      quantity: 80 
  }, {
      instanceName: "farmplane",
      quantity: 20 
  }.
  ...
]
```

File: examples/geoObject/geoObject.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+            for (let i = 0; i < COUNT; i++) {
+                entities.push(new Entity(ENTITY[name](i)));
+            }
+            geoObjects.addEntities(entities);
+        });
+};
+
+instancesTypes.forEach((type) => {
+    addInstance(type);
```
Comment (Zemledelec) [inline examples/geoObject/geoObject.js]: In terms of my previous comment, I m afraid that this makes a lot of fetches, no?

File: src/og/entity/GeoObjectHandler.js [hunk 3]
```diff
@@ -552,6 +554,64 @@ class GeoObjectHandler {
...
+            }
+            i++;
+
+        }
+
+    }
+
     //todo refactor for support instancing
     setVertexArr(index, vertexArr) {
         this._vertexArr = [-0.5, 0.0, 0.5, -0.5, 0.0, -0.5, 0.5, 0.0, -0.5, 0.5, 0.0, 0.5];
```
Comment (Zemledelec) [inline src/og/entity/GeoObjectHandler.js]: looks like useless code?

File: src/og/entity/GeoObjectHandler.js [hunk 4]
```diff
@@ -700,16 +760,16 @@ class GeoObjectHandler {
         this._indicesArr = null;
```
Comment (Zemledelec) [inline src/og/entity/GeoObjectHandler.js:754]: ...this is too

**Review by Zemledelec — CHANGES_REQUESTED** [review]
@pavletto I've made a few comments I think should be improved, or discussed first. Please take a look.

File: src/og/entity/GeoObjectHandler.js (hunk 4 above)
Comment (pavletto) [reply src/og/entity/GeoObjectHandler.js:754]: No, is not. `indicesArr` used

**Comment by pavletto** [comment]
@Zemledelec, I'm resolve

**Review by Zemledelec — APPROVED** [review]

**Pull request merged**